src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx

```diff
@@ -152,7 +152,7 @@ class GroupEventToolbar extends React.Component {
           </Link>
         </h4>
         <span>
-          <Tooltip title={this.getDateTooltip()}>
+          <Tooltip title={this.getDateTooltip()} disableForVisualTest>
```

Comment:
I think `app/utils/getDynamicText` might be preferable?

Reply:
Didn't know that existing, thanks 👍